src/app/tabs/compileTab/compilerContainer.js

```diff
@@ -319,8 +319,8 @@ class CompilerContainer {
       }
       url = `${this.data.baseurl}/${this.data.selectedVersion}`
     }
-    const isFirefox = typeof InstallTrigger !== 'undefined'
-    if (document.location.protocol !== 'file:' && Worker !== undefined && isFirefox) {
+    // Check if browser is compatible with web worker
+    if (document.location.protocol !== 'file:' && Worker !== undefined) {
```

Comment:
Maybe you can create a function for that. Like that code is self-explained, and maybe we'll need it somewhere else.
```javascript
function browserSupportWorker() {
  return document.location.protocol !== 'file:' && Worker !== undefined;
}
```

Reply:
okay sure. will add.